subprojects/docs/src/docs/userguide/definingUsingConfigurations.adoc

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[defining_and_using_configurations]]
+== Defining and Using Configurations
+
+[[sub:what_is_a_configuration]]
+=== What is a configuration?
+
+Every dependency declared for a Gradle project applies to a specific scope. For example some dependencies should be used for compiling source code whereas others only need to be available at runtime. Gradle represents the scope of a dependency with the help of a api:org.gradle.api.artifacts.Configuration[]. Every configuration can be identified by a unique name.
+
+Many Gradle plugins add pre-defined configurations to your project. The Java plugin, for example, adds configurations to represent the various classpaths it needs for source code compilation, executing tests and the like. See <<sec:java_plugin_and_dependency_management,the Java plugin chapter>> for an example. The sections above demonstrate how to <<declaring_dependencies,declare dependencies>> for different use cases.
+
++++++
+<figure xmlns:xi="http://www.w3.org/2001/XInclude">
+    <title>Configurations use declared dependencies for specific purposes</title>
+    <imageobject>
+        <imagedata fileref="img/dependency-management-configurations.png" width="200mm" />
+    </imageobject>
+</figure>
++++++
+
+For more examples on the usage of configurations to navigate, inspect and post-process metadata and artifacts of assigned dependencies, see <<working_with_dependencies>>.
```

Comment:
Can we re-organize/remove or rename working_with_dependencies yet?

Reply:
No, this has become a fully-fledged chapter with completely new content. I find the title fitting to the content except if you have better proposal.